vllm/engine/llm_engine.py

```diff
@@ -333,8 +340,10 @@ def add_request(
         # Create the sequences.
         block_size = self.cache_config.block_size
         seq_id = next(self.seq_counter)
-        eos_token_id = self.tokenizer.get_lora_tokenizer(
-            lora_request).eos_token_id
+        eos_token_id = None
+        if self.tokenizer:
+            eos_token_id = self.tokenizer.get_lora_tokenizer(
+                lora_request).eos_token_id
```

Comment:
IMO we should add a warning here - WDYT?

This will also affect components that use `eos_token_id`. For example,
https://github.com/vllm-project/vllm/blob/0ce0539d4750f9ebcd9b19d7085ca3b934b9ec67/vllm/engine/llm_engine.py#L344-L346

Reply:
I still think it's worth having an warning here about `eos_token_id` being `None`.